source/elements/oneMKL/source/domains/blas/blas-level-1-routines.rst

```diff
@@ -0,0 +1,77 @@
+.. _blas-level-1-routines:
+
+BLAS Level 1 Routines
+=====================
+
+
+.. container::
+
+
+   BLAS Level 1 includes routines and functions, which perform
+   vector-vector operations. The following table lists the BLAS Level 1
+   routine and function groups and the data types associated with them.
+
+
+   .. container:: tablenoborder
+
+
+      .. list-table:: 
+         :header-rows: 1
+
+         * -     Routines
+           -     Description     
+         * -     \ `asum <asum.html>`__\   
```

Comment:
This also needs a cleanup pass. The link to html will not work in the PDF doc.  Asum should have  a label:
.. _asum:

And then you can refer to it with :ref:`asum`

I also prefer the simple table to .. list-table::

=======  ====
Routines      Description
=======  ====
:ref:`copy`     Copy vector  
:ref:`dot`       Dot product
=======  ====

Reply:
@rscohn2 FYI, our ultimate plan is to have the onemkl tags use the prefix  "_onemkl" to avoid collisions.  An extension would be for APIs in a given onemkl domain, to add "_<domain>" that is .. _onemkl_blas_asum  would be the appropriate tag here.  And   _onemkl_blas_copy  and _oneml_blas_dot ...

Reply:
Good point, we definitely should use ref. Tables were generated with transition to rst, I think we will update them at some point as well.